front/app/translations/admin/hr-HR.json

```diff
@@ -1376,44 +1376,5 @@
-  "containers.Admin.onboarding.engageArticle3Description": "Inform your audience on a project entering a new phase, invite them for a project related event, warn them on an approaching deadline, etc.. You can do all this by creating custom email campaigns.",
-  "containers.Admin.onboarding.engageArticle3Title": "Send emails to keep your users updated",
-  "containers.Admin.onboarding.engageSectionLink": "http://support.citizenlab.co/collections/1846694-engage",
-  "containers.Admin.onboarding.engageSectionTitle": "Engage",
-  "containers.Admin.onboarding.giveFeedback": "Give feedback by adding or changing the status or providing an update.",
-  "containers.Admin.onboarding.manageArticle1Link": "/admin/projects",
-  "containers.Admin.onboarding.manageArticle1Title": "Moderate inside your projects",
-  "containers.Admin.onboarding.manageArticle2Description": "Create groups to be able to contact specific groups or define permissions per group.",
-  "containers.Admin.onboarding.manageArticle2Link": "/admin/users",
-  "containers.Admin.onboarding.manageArticle2Title": "Manage your users",
-  "containers.Admin.onboarding.manageInputs": "Manage your input",
-  "containers.Admin.onboarding.manageSectionLink": "http://support.citizenlab.co/collections/1846696-manage",
-  "containers.Admin.onboarding.manageSectionTitle": "manage",
-  "containers.Admin.onboarding.readCompleteGuide": "Read complete guide",
-  "containers.Admin.onboarding.setupArticle1Description": "Projects are the beating heart of the platform. Admins and project moderators can customize and edit the projects at any time: information, files, images, areas, timeline, participation methods, planned activities, etc...",
-  "containers.Admin.onboarding.setupArticle1Link": "/admin/projects/new",
-  "containers.Admin.onboarding.setupArticle1Title": "Create your first projects",
-  "containers.Admin.onboarding.setupArticle2Description": "You can ask your users additional information while registering. Use the registration fields to get better insights, or to segment your users into groups, for example by age, area, interests, etc...",
-  "containers.Admin.onboarding.setupArticle2Link": "/admin/settings/registration",
-  "containers.Admin.onboarding.setupArticle2Title": "Define what you want to know from your users",
-  "containers.Admin.onboarding.setupArticle3Description": "A personalized widget brings people from other websites to your platform.",
-  "containers.Admin.onboarding.setupArticle3Link": "/admin/settings/widgets",
-  "containers.Admin.onboarding.setupArticle3Title": "Link the platform to third party websites using a widget",
-  "containers.Admin.onboarding.setupSectionLink": "http://support.citizenlab.co/collections/1846692-set-up",
-  "containers.Admin.onboarding.setupSectionTitle": "Setup",
-  "containers.Admin.onboarding.title": "Guide"
+  "components.admin.ServerError.genericServerError": "There was an issue loading this content, please try again later."
```

Comment:
I wonder why we have this diff - as far as I can tell, no translations were made with this PR.

Reply:
That came from the CI's auto commit here: https://github.com/CitizenLabDotCo/citizenlab/pull/1821/commits/54f4da61cf60140a5385ecac04f0146400559d57

Why? No idea. But the computer did it 😅